pkg/scheduler/framework/framework_test.go

```diff
@@ -4540,7 +4540,7 @@ func TestRunAllPluginsForPickNPlacementType(t *testing.T) {
 	dummyScorePluginNameA := fmt.Sprintf(dummyAllPurposePluginNameFormat, 6)
 	dummyScorePluginNameB := fmt.Sprintf(dummyAllPurposePluginNameFormat, 7)
 
-	clusters := []fleetv1beta1.MemberCluster{
+	clusters := []clusterv1beta1.MemberCluster{
```

Comment:
Are those additional changes needed after the CRD split? Will they disappear after rebase?

Reply:
Yeah, they will disappear after rebasing. I am rebasing the PR now.